.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Completed each of the following items before creating a PR
+
```

Comment:
I thought about that but we check for a ticket number in the commit message. I'm not sure how Github will push that commit message into the template.